src/summarease/__init__.py

```diff
@@ -1,3 +1,3 @@
 # read version from installed package
 from importlib.metadata import version
-__version__ = version("summarease")
\ No newline at end of file
+__version__ = version= ("summarease")
```

Comment:
I run with this and it works, though I'm not sure this line is legal python so we may not want to include it 
![image](https://github.com/user-attachments/assets/f3ab5064-4fe1-4fda-8cd9-aa46205c84dd)